site/content/docs/user/quick-start.md

```diff
@@ -147,6 +147,15 @@ See [Kubernetes imagePullPolicy][Kubernetes imagePullPolicy] for more informatio
 
 See also: [Using kind with Private Registries][Private Registries].
 
+**Note**:
+if you got `Failed to pull image "my-custom-image:unique-tag"` after applying your manifest. You can verify if kind pulled your image into its cluster by calling`crictl images` command inside `kind-control-plane`. 
+
+```
+docker exec -it --privileged  kind-control-plane /bin/sh
```

Comment:
nit: shouldn't need `--privileged` or `/bin/sh`?

Reply:
Not sure I understood your comment here 

(base) azadehkhojandi@AzadehKhojandi:/mnt/c/Lab/go/src/Telstra.Dx.AzureOperator$ docker exec -it --privileged  kind-control-plane /bin/sh
# crictl images
IMAGE                                      TAG                 IMAGE ID            SIZE
docker.io/azadehkhojandi/azoperator        v1                  c3c21f4f4d43d       50.1MB
docker.io/azadehkhojandi/azoperator        v2                  e5cde5eec42a2       50.1MB
docker.io/azadehkhojandi/azoperator        v3                  ac26c77753c27       50.1MB
docker.io/kindest/kindnetd                 0.5.0               ef97cccdfdb50       83.6MB
gcr.io/kubebuilder/kube-rbac-proxy         v0.4.0              8323481d90856       16MB
k8s.gcr.io/coredns                         1.3.1               eb516548c180f       40.5MB
k8s.gcr.io/etcd                            3.3.10              2c4adeb21b4ff       258MB
k8s.gcr.io/kube-apiserver                  v1.15.0             28bdc6f28a8d9       249MB
k8s.gcr.io/kube-controller-manager         v1.15.0             4ccdc592ea9c4       200MB
k8s.gcr.io/kube-proxy                      v1.15.0             241709fdde11b       97.3MB
k8s.gcr.io/kube-scheduler                  v1.15.0             e855173d0e51e       96.5MB
k8s.gcr.io/pause                           3.1                 da86e6ba6ca19       746kB
quay.io/jetstack/cert-manager-cainjector   v0.8.1              99caccfb0d1fc       13.7MB
quay.io/jetstack/cert-manager-controller   v0.8.1              30462d9059d35       16.1MB
quay.io/jetstack/cert-manager-webhook      v0.8.1              f88235cd97063       16.1MB
quay.io/solo-io/plank-dlv                  0.5.17              6277c693a9aa1       321MB
# exit
(base) azadehkhojandi@AzadehKhojandi:/mnt/c/Lab/go/src/Telstra.Dx.AzureOperator$ docker exec -it  kind-control-plane"docker exec" requires at least 2 arguments.
See 'docker exec --help'.

Usage:  docker exec [OPTIONS] CONTAINER COMMAND [ARG...]

Run a command in a running container

Reply:
maybe Ben means 


docker exec -it  kind-control-plane crictl images